sigh/bin/sigh

```diff
@@ -67,6 +68,7 @@ class SighApplication
                'The path may be prefixed with a identifier in order to determine which provisioning profile should be used on which app.',
                &multiple_values_option_proc(c, "provisioning_profile", &proc { |value| value.split('=', 2) })
       c.option '-d', '--display_name STRING', String, 'Display name to use'
+      c.option '-g', '--new_bundle_id STRING', String, 'New application bundle ID'
```

Comment:
Could you use `-b` for short option here?
As far as I can tell, `-b` is not yet used and is available. It's also more related, while `b` -> `b`undle is easy to remember, `g` is a bit cryptic.